lib/src/Uncertainty/Distribution/MeixnerDistribution.cxx

```diff
@@ -422,19 +422,20 @@ void MeixnerDistribution::update()
 
   // Initilalyse Optimization problems
   OptimizationProblem problem;
-  problem.setBounds(getRange());
-  solver_.setStartingPoint(getMean());
 
   // Define Optimization problem1 : maximization fB
   problem.setMinimization(false);
   problem.setObjective(fB);
+  problem.setBounds(getRange());
+  solver_.setStartingPoint(getMean());
   solver_.setProblem(problem);
   solver_.run();
   b_ = std::sqrt(solver_.getResult().getOptimalValue()[0]);
 
   // Define Optimization problem2 : minimization fCD
   problem.setMinimization(true);
   problem.setObjective(fCD);
+  problem.setBounds(getRange());
```

Comment:
No need to set the bounds another time if the setObjective() method is modified according to my proposal